test/cases/dml/select/select.test

```diff
@@ -186,3 +186,8 @@ CREATE TABLE t1 (c0 varchar(0) DEFAULT NULL);
 insert into t1 values();
 insert into t1 values('');
 select * from t1;
+
+drop table if exists t1;
+create table if not exists t1(a int auto_increment);
```

Comment:
1、one auto increment column in on table.
2、auto increment column must a key(PK or UK)
you can add these test case